lib/squib/graphics/save_sprue.rb

```diff
@@ -158,11 +160,21 @@ class SaveSpruePDF < SaveSprue
       def init_cc
         ratio = 72.0 / @deck.dpi
 
-        surface = Cairo::PDFSurface.new(
-          full_filename,
-          @tmpl.sheet_width * ratio,
-          @tmpl.sheet_height * ratio
-        )
+        slots = @tmpl.cards
+        per_sheet = slots.size
+	      if per_sheet == 1
```

Comment:
Can you change this if-block to this construction?

```ruby
surface = if per_sheet ==1
  Cairo::PDFSurface.new(
  #...
 )
#...
end
```

Reply:
Change it according to your suggestion. And added the change in the save_pdf DSL doc under the sprue option.